VelocitySDK/Sensors/Recorders/VLTGPS.m

```diff
@@ -0,0 +1,36 @@
+//
+//  VLTGPS.m
+//  VelocitySDK
+//
+//  Created by Vytautas Galaunia on 10/08/2017.
+//  Copyright © 2017 Veloctity. All rights reserved.
+//
+
+#import "VLTGPS.h"
+#import "VLTUserDataStore.h"
+
+NSString * const VLTGPSLocationUpdateNotification = @"VLTGPSLocationUpdateNotification";
+NSString * const VLTGPSLocationsKey = @"VLTGPSLocationsKey";
+
+@implementation VLTGPS
+
++ (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray<CLLocation *> *)locations
```

Comment:
As it is an SDK, I would like to keep it here. Because it can really happen than we are going to need it.